packages/manager/apps/pci-public-ip/src/components/list/FloatingIP.tsx

```diff
@@ -103,14 +115,88 @@ export default function FloatingIPComponent({ projectId, projectUrl }) {
           />
           {t('pci_additional_ips_add_additional_ip')}
         </OsdsButton>
+        <div className="justify-between flex">
+          <OsdsSearchBar
+            className={'w-[70%]'}
+            value={searchField}
+            onOdsSearchSubmit={({ detail }) => {
+              addFilter({
```

Comment:
i think you need to add :

setPagination({
  pageIndex: 0,
  pageSize: pagination.pageSize,
});

otherwise it causes issues if you are doing search being already on 2nd page